apps/accounts/serializers/account/account.py

```diff
@@ -385,7 +385,7 @@ def perform_bulk_create(self, vd):
 
         _results = {}
         for asset in assets:
-            if asset not in secret_type_supports:
+            if asset not in secret_type_supports and asset.category != Category.CUSTOM:
                 _results[asset] = {
                     'error': _('Asset does not support this secret type: %s') % secret_type,
                     'state': 'error',
```

Comment:
There do not appear to be any significant bugs or anomalies in the provided Python code.

The current version of the code seems appropriate for its intended purpose. However, there is an unusual character before `for` at the start of the first loop:

```
_results = {}

```

It should probably read like this:

```
_results = {}  
```